src/components/application_manager/test/CMakeLists.txt

```diff
@@ -126,15 +184,25 @@ set(LIBRARIES
   formatters
   utils
   resumption
+  dbms
   jsoncpp
 )
 
+create_test(application_test "${APPLICATION_SOURCES}" "${LIBRARIES}")
+create_test(application_state_test "${APPLICATION_STATE_SOURCES}" "${LIBRARIES}")
+create_test(application_manager_test "${APPLICATION_MANAGER_SOURCES}" "${LIBRARIES}")
 create_test(am_commands_test "${COMMANDS_SOURCES}" "${LIBRARIES}")
 create_test(am_message_helper_test "${MESSAGE_HELPER_SOURCES}" "${LIBRARIES}")
 create_test(am_request_controller_test "${REQUEST_CONTROLLER_SOURCES}" "${LIBRARIES}")
 create_test(am_resumption_test "${RESUMPTION_SOURCES}" "${LIBRARIES}")
 create_test(am_state_controller_test "${STATE_CONTROLLER_SOURCES}" "${LIBRARIES}")
-create_test(application_manager_test "${SOURCES}" "${LIBRARIES}")
+create_test(am_mobile_controller_test "${MOBILE_MESSAGE_SOURCES}" "${LIBRARIES}")
+create_test(am_event_engine_test "${AM_EVENT_ENGINE_SOURCES}" "${LIBRARIES}")
+create_test(am_hmi_capabilities_test "${AM_HMI_CAPABILITIES_SOURCES}" "${LIBRARIES}")
+create_test(am_policy_handler_test "${AM_POLICY_HANDLER_SOURCES}" "${LIBRARIES}")
+create_test(am_policy_event_observer_test "${AM_POLICY_EVENT_OBSERVER_SOURCES}" "${LIBRARIES}")
+create_test(am_usage_statistics_test "${AM_USAGE_STATISTICS_SOURCES}" "${LIBRARIES}")
+create_test(am_zero_request_amount_test "${AM_ZERO_REQUEST_AMOUNT_SOURCES}" "${LIBRARIES}")
```

Comment:
@AGritsevich I agree with @anosach-luxoft  at least I don't understand what the purposu of the latest test `am_zero_request_amount_test` looks like it could be part of some other test